### PR TITLE
Add script to automatically add contributor list to readme.md

### PR DESCRIPTION
**Pull request by fraindz**
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)

* **What kind of change does this PR introduce? 
This PR is for doc update.

> yarn contributors:addAll

This will automatically add thumbnails of project contributors to readme.md
…

* **Does this close any currently open issues?**
#180 
…

* **What is the new behavior (if this is a feature change)?**
No feature change. This is for better documentation.
…

* **Where has this been tested?**

    * Operating System: …
      Windows 10

### CODE REVIEW

**Comment by fraindz** [comment]
@btzr-io Please review the PR

**Comment by btzr-io** [comment]
@fraindz Hi, thanks for your contribution, Could you make this a grid showing the user thumbnails?

**Comment by fraindz** [comment]
@btzr-io Sure, that will look wonderful. It might take some more time though...

**Comment by fraindz** [comment]
Added thumbnails of existing contributors.

To add all new contributors automatically to readme, run below script

> yarn contributors:addAll

File: README.md [hunk 1]
```diff
@@ -64,3 +64,14 @@ $ yarn dist
...
+| [<img src="https://avatars2.githubusercontent.com/u/1966512?v=4" width="100px;"/><br /><sub><b>Rogelio Orts</b></sub>](http://rogelioorts.com)<br />[💻](https://github.com/btzr-io/jelly-beats/commits?author=rogelio-o "Code") |
```
[the remark this answers is not in the document]
Comment (fraindz) [reply README.md]: The image in question here is auto-generated.

**Comment by fraindz** [comment]
image
This is how it will look post merge.

**Comment by fraindz** [comment]
@btzr-io Updated code to show thumbnails of contributors. Please review.

**Comment by btzr-io** [comment]
@fraindz Could you resolve the conflicts so I can merge this?

**Comment by fraindz** [comment]
@btzr-io Merge conflicts are resolved now.

**Pull request merged**